### DISCUSSION TITLE
How to connect to Mullvad Ashburn VA servers?

### DISCUSSION THREAD

**Discussion by calebsw83** [qdm12/gluetun#902] [Q&A] [answered]
It seems like Mullvad recently added Ashburn VA servers, however, ashburn is not one of the allowable cities in the gluetun config. I tried omitting the server cities flag, and just using the private key and address for wireguard, but that seems to randomly selects a location for me. Is there any way to use Ashburn servers currently? Is there a way for gluetun to not need to be manually updated every time any new locations that mullvad gets added?

**Comment by qdm12** [comment, marked as answer]
The latest image probably has updated Mullvad information, so you can try that.

Otherwise see https://github.com/qdm12/gluetun/wiki/Updating-servers to update the Mullvad servers yourself :wink: That should add the ashburn servers

**Comment by calebsw83** [comment]
Oh wow I forgot all about this thread. I just resigned myself to using the next closest server indefinitely so this is quite a pleasant way to start the week! 

Maybe I just have the dumb, but I can't seem to get this to work. I normally stay on numbered versions because it breaks my stack whenever I'm on `latest` and it updates (it used to update daily, maybe not the case anymore. Anyways, I updated to `latest` and that didn't allow connection to Ashburn, and I tried to manually updating the servers, and I got an error that said:

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x702571]
```

I tried updating servers.json manually, and I'm pretty sure I did it right, but when I run gluetun, it either recreates it, or it pulls a fresh one from the web. I tried setting the file and folder to read only, but still no luck. It still tells me that Ashburn is not a valid choice in the environment variables. 

I'm not very experienced with linux, so I might have messed that up. I copied the public key from each server from the config files I got from mullvad, so I would think it would at least attempt to connect. IDK the ISP or if they are self-owned though, not sure if that has any bearing on it. Not sure what I'm doing wrong. 

I'll probably just have to wait until these officially get added. If I can help with that somehow, I'd be glad to assist.
